linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_bashrc_tmux/rule.yml

```diff
@@ -0,0 +1,79 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Support session locking with tmux (allow leaving Tmux)'
```

Comment:
How about the following suggestion? Not quite sure if it improves anything.

```suggestio
title: 'Support session locking with tmux (not enforcing)'
```

Reply:
I like the suggestion, I don't think I will come up with something better.